_data/identity.yml

```diff
@@ -6,6 +6,19 @@ websites:
       software: Yes
       doc: https://support.1password.com/two-factor-authentication/
 
+    - name: Auth0
+      url: https://auth0.com/
+      img: auth0.png
+      tfa: Yes
+      sms: Yes
+      email: Yes
+      phone: Yes
```

Comment:
Could you point me to where in the doc phone calls are mentioned?

Reply:
Just checking on this, I believe through a 3rd party only.